README.md

```diff
@@ -61,6 +61,7 @@ SYSTEM_USER_ENABLED (by default is `true`).
 | SYSTEM\_USER\_ENABLED      | true                            | Defines if system user must be created at service tenant initialization or used for egress service requests |
 | FQM\_MAX\_CHUNK\_SIZE      | 10000                           | Max chunk size of FQM Fetcher                                                                               |
 | FQM\_MAX\_PARALLEL\_CHUNKS | 10                              | Max number of parallel chunks processed at the same time                                                    |
+| FQM_APPROACH               | false                           | Approach to use with query: false if identifiers flow, otherwise query flow                                 |
```

Comment:
FQM\_APPROACH - it’s generally safer to escape underscores (\_) in markdown file.

Reply:
I would also suggest to rename this parameter to QUERY_BASED_APPROACH.

Reply:
Maybe it would be better to rename it to FQM_QUERY_APPROACH? Because actually we disable fqm in query, not query on its own.

Reply:
Agreed.